crates/starknet-server/tests/common/constants.rs

```diff
@@ -34,3 +34,6 @@ pub const CAIRO_1_PANICKING_CONTRACT_SIERRA_PATH: &str =
 // Contract which asserts the tx version
 pub const CAIRO_1_VERSION_ASSERTER_SIERRA_PATH: &str =
     "test_data/cairo1/version_asserter/version_asserter.sierra";
+
+// L1 L2 pre-registered addresses.
+pub const MESSAGING_L1_ALLOWED_CONTRACT: &str = "0x8359e4b0152ed5a731162d3c7b0d8d56edb165a0";
```

Comment:
I remember seeing this was addressed. It would be good to have the commit link here, but I don't remember which one it was, so never mind, I'll just resolve this conversation.